src/components/MediaPlayer.vue

```diff
@@ -103,12 +108,18 @@ export default {
       return `${displayHours}${displayMinutes}:${displaySeconds}`;
     },
     changePlaybackRate() {
-      if (this.player.playbackRate === 2) {
-        this.player.playbackRate = 0.5;
+      const playbackRateSpeeds = {
+        0.5: 0.8,
+        0.8: 1,
+        1: 1.2,
+        1.2: 1.5,
+        1.5: 2
```

Comment:
let's add a 1.8 speed between 1.5 and 2.0. We probably don't talk slowly enough for it to be useful, but i definitely listen to audio books at 1.75x speed, so might as well have it here too.

Reply:
Done